pkg/kube/client.go

```diff
@@ -687,14 +687,52 @@ func updateResource(c *Client, target *resource.Info, currentObj runtime.Object,
 		c.Log("Patch %s %q in namespace %s", kind, target.Name, target.Namespace)
 		obj, err = helper.Patch(target.Namespace, target.Name, patchType, patch, nil)
 		if err != nil {
-			return errors.Wrapf(err, "cannot patch %q with kind %s", target.Name, kind)
+			sanitizeLog:=err.Error()
```

Comment:
please format the code.

Reply:
sure, I'll update this code in the next commit.